### DISCUSSION TITLE
Is adding a copy of a pre-existing field instead of editing it expected behavior?

### DISCUSSION THREAD

**Discussion by artjafri** [RafaelGB/obsidian-db-folder#140] [Q&A]
I've run into an issue where If I have an inline field called "Type" with the value "Book", and I create a column also called "Type" it will automatically pull in and show the value "Book", which already exists.

But if I change "book" to "movie", the plug in will instead create a new field called "Type" at the top of the file, with the value "movie".

on plugin/page reload the column will display "movie book". not sure if this expected behavior or im doing something wrong

**Comment by RafaelGB** [comment]
Hi @artjafri 
actually the inline fields support has the next pattern:
```md
MyField:: value
```

If your inline field is declared with bold, into a list of simply not at start of the line, the plugin its not smart enough to detect that exist and just create a new one

inline not supported YET
```md
**myField**:: value

- myField:: value

this is a sample text with myField:: value
```

This is the issue of the roadmap to support them https://github.com/RafaelGB/obsidian-db-folder/issues/134
